package.json

```diff
@@ -90,7 +90,7 @@
     "mc_dir": "../mozilla-central"
   },
   "scripts": {
-    "mochitest": "(cd $npm_package_config_mc_dir && ./mach mochitest browser/extensions/activity-stream/test/functional/mochitest )",
+    "mochitest": "(cd $npm_package_config_mc_dir && ./mach mochitest browser/extensions/activity-stream/test/functional/mochitest --headless )",
```

Comment:
can you edit the `yamscripts` file instead? Also, would it be worth it / possible to to run `browser/components/newtab/`  and maybe also `./mach test all_files_referenced | grep activity-stream`?

Reply:
oops true.